### PR TITLE
ENT-4901 create ssl dir on Windows

### PR DESCRIPTION
**Pull request by Lex-2008**
Merge together with:
https://github.com/cfengine/buildscripts/pull/590

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
testbuild: https://ci.cfengine.com/job/pr-pipeline/2570/

**Comment by Lex-2008** [comment]
SUCCESS build: https://ci.cfengine.com/job/pr-pipeline/2573/

File: packaging/cfengine-nova/cfengine-nova.wxs [hunk 1]
```diff
@@ -141,6 +141,12 @@
 	      <Component Id='libcurl_4.dll' Guid='de435dd1-1fce-424a-b6a0-dc63192e2060' Win64='$(var.isWin64)'>
                 <File Id='libcurl_4.dll' Name='libcurl-4.dll' KeyPath='yes' DiskId='1' Source='$(var.CfSourceDir)\bin\libcurl-4.dll' />
               </Component>
+            </Directory>
+			<!-- ===== Cfengine\ssl ===== -->
+            <Directory Id='dir_ssl' Name='ssl'>
+              <Component Id='openssl.cnf' Guid='51EF6C21-D0AD-4A29-9F83-B72B5DF614A5' Win64='$(var.isWin64)'>
+                <File Id='openssl.cnf' Name='openssl.cnf' KeyPath='yes' DiskId='1' Source='$(var.CfSourceDir)\ssl\openssl.cnf' />
+              </Component>
```
Comment (vpodzime) [inline packaging/cfengine-nova/cfengine-nova.wxs]: There should be more files in the directory, AFAICT.

**Comment by Lex-2008** [comment]
above build looks nice!

File: packaging/cfengine-nova/cfengine-nova.wxs (hunk 1 above)
Comment (Lex-2008) [reply packaging/cfengine-nova/cfengine-nova.wxs]: Which files?
after `cp -a /var/cfengine/ssl/* /home/jenkins/workspace/testing-pr/label/PACKAGES_i386_mingw/./packaging/cfengine-nova/pkg/testing-pr/ssl` command, there is only this one file in in that dir:
```
18:58:43 + ls -la /home/jenkins/workspace/testing-pr/label/PACKAGES_i386_mingw/./packaging/cfengine-nova/pkg/testing-pr/ssl
18:58:43 total 20
18:58:43 drwxrwxr-x 2 jenkins jenkins  4096 Aug 12 16:58 .
18:58:43 drwxrwxr-x 4 jenkins jenkins  4096 Aug 12 16:58 ..
18:58:43 -rw-r--r-- 1 jenkins jenkins 10909 Jul 16 12:08 openssl.cnf
```
see https://ci.cfengine.com/job/testing-pr/3442/label=PACKAGES_i386_mingw/console for full output (note that the command in the log was expanded from `cp -a /var/cfengine/ssl/* …`)

Comment (vpodzime) [reply packaging/cfengine-nova/cfengine-nova.wxs]: Interesting. `/var/cfengine/ssl` contained more files when I checked it on a CentOS 6, but maybe those are created later? Anyways, this should be OK then. :+1:

**Comment by vpodzime** [comment]
A build with `/merge` not `/head`:
[Build Status](https://ci.cfengine.com/job/pr-pipeline/2579/)

**Pull request merged**